test/schedule_run_once_test.js

```diff
@@ -37,6 +37,32 @@ describe('Scheduled job with interval', ()=> {
       it('should run the job once', ()=> {
         assert.equal(count, 1);
       });
+    });
+
+    describe('after running job once', ()=> {
+
+      before(()=> {
+        TimeKeeper.travel(Date.now() + ms('30m'));
+      });
+
+      before(Ironium.runOnce);
+
+      it('should not run job again before next occurrence', ()=> {
+        assert.equal(count, 1);
+      });
+    });
+
+    describe('after second occurence', ()=> {
```

Comment:
This test would fail for a different reason. If we start the clock at 10:10 and run this test, the clock is now 10:41, so we didn't cross the hour mark and run the hourly job. This test will only succeed if it runs within the context of the previous test.